features/admin.applications.v1/components/settings/sign-on-methods/sign-in-method-customization.tsx

```diff
@@ -177,6 +178,11 @@ export const SignInMethodCustomization: FunctionComponent<SignInMethodCustomizat
     const [ smsValidationResult, setSmsValidationResult ] =
         useState<FederatedConflictWithSMSOTPReturnValueInterface>(null);
 
+    const sharedAppAdaptiveAuthDisabled: boolean = useSelector((state: AppState) => {
+        return state.config?.ui?.features?.applications?.disabledFeatures?.includes(
+            SHARED_APP_ADAPTIVE_AUTH_FEATURE_ID);
+    });
```

Comment:
Let's use `isFeatureEnabled` method from core module for better readability. and also let's keep the variables in positive form if possible, for better readability.

```suggestion
    const sharedAppAdaptiveAuthEnabled: boolean = useSelector((state: AppState) => {
        return isFeatureEnabled(state.config?.ui?.features?.applications, SHARED_APP_ADAPTIVE_AUTH_FEATURE_ID);
    });
```

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/9137/commits/d4511ae891f6ef0237edfa30b73041a2f3ab2dbb